fs/dafsserver/dafsserver.cpp

```diff
@@ -2278,6 +2278,7 @@ class CRemoteIndexReadActivity : public CRemoteIndexBaseActivity
 
     Owned<const IDynamicTransform> translator;
     unsigned __int64 chooseN = 0;
+    bool cleanupBlobs = false;
```

Comment:
is it necessary to check in dtor in case of exception? or will they be cleared up as the index is released anyway?